lib/bandit/pipeline.ex

```diff
@@ -225,17 +229,20 @@ defmodule Bandit.Pipeline do
     {:error, error}
   end
 
-  defp handle_error(error, stacktrace, transport, span, opts) do
-    Bandit.Telemetry.span_exception(span, :exit, error, stacktrace)
-    status = error |> Plug.Exception.status() |> Plug.Conn.Status.code()
+  defp handle_error(kind, reason, stacktrace, transport, span, opts) do
+    Bandit.Telemetry.span_exception(span, telemetry_kind(kind), reason, stacktrace)
```

Comment:
Had to add this `telemetry_kind/1` function to keep existing behavior of emitting `:exit` kind on an exception.

Feels like this should be `:error` instead, and `:exit` would be if we later also emit telemetry event for non-normal exits by augmentig the `catch` to also...

```elixir
catch
  :exit, reason ->
```

Reply:
> Had to add this `telemetry_kind/1` function to keep existing behavior of emitting `:exit` kind on an exception.

Solved this in the newer PR, by introducing a new telemetry event and keeping the original one for backwards compatibility: https://github.com/mtrudel/bandit/pull/398/files#diff-68626306df11d3e10964941997c6cdafbf587a9ec7de087141840655a47a1855R231-R236